exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Configuration.java

```diff
@@ -48,4 +58,44 @@
    */
   <MessageT extends MessageLite> MessageT getAsMessage(Class<MessageT> parametersType);
 
+  /**
+   * Returns the configuration as a plain text string.
+   *
+   * @throws IllegalArgumentException if the actual type of the configuration is not an instance of
+   *     {@link ServiceConfiguration};
+   *     or the configuration is not in the {@linkplain Format#TEXT plain text} format
```

Comment:
I'd allow to get any format as text (e.g., a service needs a fancier Gson configuration).